Core/source/core/pgp-key.ts

```diff
@@ -297,9 +302,9 @@ export class PgpKey {
     return Boolean(await key.getEncryptionKey(undefined, oneSecondBeforeExpiration));
   }
 
-  public static dateBeforeExpiration = async (key: OpenPGP.key.Key | string): Promise<Date | undefined> => {
+  public static dateBeforeExpiration = async (key: Key | string): Promise<Date | undefined> => {
     const openPgpKey = typeof key === 'string' ? await PgpKey.read(key) : key;
-    const expires = await openPgpKey.getExpirationTime('encrypt');
+    const expires = await openPgpKey.getExpirationTime();
```

Comment:
Is this no longer supported?

Reply:
Need to ask maintainers

Reply:
Asked question https://github.com/openpgpjs/openpgpjs/discussions/1493

Reply:
At least there is no more such parameter.

Reply:
@tomholub I've backported old function from openpgp v4

Reply:
ok for now